Mobile/ShopLens/ShopLens/Extensions/ExtensionMethods.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace ShopLens.Extensions
+{
+    static class ExtensionMethods
+    {
+        public static string FirstCharToUpper(this string input)
+        {
+            switch (input)
+            {
+                case null:
+                    throw new ArgumentNullException(nameof(input));
+
+                case "":
+                    throw new ArgumentException($"{nameof(input)} cannot be empty", nameof(input));
+
+                default:
+                    return input.First().ToString().ToUpper() + input.Substring(1);
```

Comment:
By the way, this class is just begging :pray: to be tested :heavy_check_mark: / :x: 

Like here, I'm not sure if `.Substring(1)` will throw an exception if the string is just 1 letter?

(I think that the docs say it won't throw in this case actually https://docs.microsoft.com/en-us/dotnet/api/system.string.substring?view=netframework-4.7.2)

Again, just want you to start spotting these places where unit tests are great :eyes:

Reply:
Ok, so I should write a Unit test? Again, not my code here.